doc/api/net.md

```diff
@@ -2,9 +2,14 @@
 
 > Stability: 2 - Stable
 
-The `net` module provides you with an asynchronous network wrapper. It contains
-functions for creating both servers and clients (called streams). You can include
-this module with `require('net');`.
+The `net` module provides an asynchronous network API for creating stream-based
+servers ([`net.Server`][]) and clients ([`net.Socket`][]) that implement TCP
```

Comment:
Most of the examples for `net.Socket` call the returned value `client` though...? I guess saying it provides clients would help people looking for one.